app/services/doc_auth_router.rb

```diff
@@ -163,6 +166,7 @@ def self.client(warn_notifier: nil)
           sharpness_threshold: IdentityConfig.store.doc_auth_error_sharpness_threshold,
           glare_threshold: IdentityConfig.store.doc_auth_error_glare_threshold,
         ),
+        vendor_discriminator: vendor_discriminator,
```

Comment:
Alternatively could we just pass the vendor verbatim? (Would require a couple more updates)

```suggestion
        vendor: 'acuant',
```

Or could it retrieve vendor based on `@client.is_a?(DocAuth::Acuant::AcuantClient)`

Reply:
I'm not sure I understand this one. The value passed here is the document capture session uuid not the vendor name itself.

Reply:
In the class, the only thing we use it for is this condition: 

https://github.com/18F/identity-idp/blob/ca83ad3f000376e0d7c21fa3ba4e38cbc7daefe8/app/services/doc_auth_router.rb#L120-L122

So the main thing we care about is the vendor? Rather than mapping it back from the vendor discriminator, we already know the vendor we're switching on at this code (`when 'acuant'`).

Or it could be inferred by client, e.g.

```rb
 error_keys.delete(:selfie) if @client.is_a?(DocAuth::Acuant::AcuantClient)
```

Reply:
Ah! Ok. I see what you're saying. Yeah, that should simplify this part. The information is already available elsewhere.

Reply:
Wait, since we wrap the doc auth clients in a translation proxy, I don't think that check will work?

Maybe we should add `Client#vendor_name` or something since that would be clearer?

Reply:
@zachmargolis I just double checked. This check is happening inside `DocAuthErrorTranslatorProxy` so `@client' in this scope is the actual vendor client, not the wrapper. 

I also checked actually running the code just to make sure something weird wasn't happening but it's working as expected.

Reply:
ah perfect, I didn't realize it was inside the proxy 👍